AndroidGesture/app/src/main/java/com/example/ben/androidgesture/PebbleReceiver.java

```diff
@@ -162,4 +170,29 @@ public void run() {
         }, 2000);
     }
 
+    public void onRadioButtonClicked(View view) {
```

Comment:
Ideadlly, we ll use this later for some other thing i.e. which device to target maybe.